espflash/src/flasher/mod.rs

```diff
@@ -153,7 +154,7 @@ impl FlashSize {
 
 impl FromStr for FlashSize {
     type Err = Error;
-
+    /// Create a `FlashSize` from a string
```

Comment:
```suggestion
    /// Create a [FlashSize] from a string
```

There may be other instances like this, but I'll just point out this one: we can link to internal types instead, which just makes the documentation a bit nicer to use IMO